lesson_2/homework/calculate_time.py

```diff
@@ -0,0 +1,36 @@
+from random import randint
+
+
+def calculate_time(n: int) -> tuple[int, int, int, int]:
+    """
+    Calculates number of full minutes since the beginning of the day,
+    number of full hours since the beginning of the day,
+    number of seconds since the beginning of the last minute,
+    number of minutes since the beginning of the last hour.
+
+    Args:
+        n (int): The number of seconds.
+
+    Returns:
+        tuple[int, int, int, int]: Resulting numbers.
+    """
+    if not isinstance(n, int):
+        raise ValueError('Object n must be integer')
+    if not 0 <= n <= 86400:
```

Comment:
Use wise versa condition 
```suggestion
    if 86400 <= n <= 0:
```

Reply:
Okay, understood